src/share/util/shr_pio_mod.F90

```diff
@@ -177,7 +182,18 @@ subroutine shr_pio_init2(comp_id, comp_name, comp_iamin, comp_comm, comp_comm_ia
 
     if(pio_async_interface) then
 #ifdef PIO1
-       call pio_init(total_comps,mpi_comm_world, comp_comm, io_comm, iosystems, rearr_opts=pio_rearr_opts)
+       call pio_init(total_comps,mpi_comm_world, comp_comm, io_comm, iosystems)
```

Comment:
We only have the "#ifdef PIO1" around the async interface. In PIO2 the async interface does not work in Fortran (for a while Ed was working on changing the async C interface, but I think now we should be able to implement the async fortran interface in pio2)